controls/entry/entry-overview.md

```diff
@@ -8,17 +8,19 @@ slug: entry-overview
 
 # Overview
 
-**RadEntry** is a text input control, which accepts string input from the users and provides consistent look and feel with the rest of the Telerik UI for Xamarin components through innovative theming mechanism.
+**RadEntry** is a text input control that accepts string input and provides a look and feel consistent with the rest of the Telerik UI for Xamarin components through an innovative theming mechanism.
+
+If you are new to RadEntry, see the [Getting Started]({% slug entry-getting-started%}) guide that demonstrates how to add the control to your application.
 
 #### Figure 1: RadEntry Overview
 ![Entry Overview](images/entry_overview.png "Entry Overview")
 
 ## Key features
 
- * **Watermark**: RadEntry allows setting a placeholder text in the control, which could hint the users what is the input that is expected. For more details read [here]({% slug entry-key-features %}#watermark).
- * **Password functionality**: You could hide the characters a user is typing for privacy reasons, check [here]({% slug entry-key-features %}#password) for more details.
- * **Border styles**: RadEntry gives you the option to customize the look of the border around the input using the BorderStyle property, for more details read [here]({% slug entry-key-features %}#borderstyle).
- * **Keyboard support**: You could specify the virtual keyboard to be used with RadEntry considering the expected input.
+* **Watermark**: RadEntry allows you to add a hint text as a placeholder in the control. You can use this text to guide the users what is the expected input. For more details, see the [Key Features]({% slug entry-key-features %}#watermark) article.
```

Comment:
Try to use descriptive links, for example, _Key Features_ instead of _here_.

Reply:
Agree